src/otelcollector/otelcol-config-extras.yml

```diff
@@ -23,13 +23,24 @@ exporters:
 processors:
   attributes:
     actions:
+      - action: insert
+        key: service.name
```

Comment:
duplicate with `service_name` below?

Reply:
There will be no duplicates in the final log record.
`service.name`, `service.instance.id` will be moved to resource attributes (`groupbyattrs` will do this).

In the end, the log record will have:
labels:  `service_name`, `job`, `instance`
attributes: `container_name`, `container_id`
resource.attributes: `service.name`, `service.instance.id`

We can remove `service_name` attribute if we decide to go with `job`

Reply:
I mean we have
```
      - action: insert
        key: service.name
        from_attribute: container_name
```
and then
```
      - action: insert
        key: service_name
        from_attribute: container_name
```

Reply:
got it, thank you!